src/main/scala/beam/agentsim/infrastructure/charging/ChargingPointType.scala

```diff
@@ -116,7 +122,13 @@ object ChargingPointType {
       case ChargingStationCcsComboType2 => DC
       case TeslaSuperCharger            => DC
       case CustomChargingPoint(_, _, c) => c
-      case _                            => throw new IllegalArgumentException("invalid argument")
+      // legacy charging points
+      case Level2    => AC
+      case Level1    => AC
+      case DCFast    => DC
+      case UltraFast => DC
+      case NoCharger => AC
```

Comment:
doesn't make sense as a call on NoCharger should throw an IllegalArgumentException but we can keep this anyway for now to make it work.